src/index.tsx

```diff
@@ -1,17 +1,52 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import './index.css'
-import App from './App'
-import reportWebVitals from './reportWebVitals'
+import firebase from 'firebase/compat/app'
+import 'firebase/compat/auth'
+import 'firebase/compat/firestore'
+import { createStore } from 'redux'
+import { Provider } from 'react-redux'
+import { ReactReduxFirebaseProvider } from 'react-redux-firebase'
+import { createFirestoreInstance } from 'redux-firestore'
+import { rootReducer } from './store/reducers'
+import { AppRouter } from './routers/AppRouter'
+
+const firebaseConfig = {
```

Comment:
It would be great to keep `index.tsx` file smaller and do not keep here configuration related to third-party libraries. Consider moving Firebase and store configs to separate files.

Reply:
I will make separate files for firebase and store config